debian/control

```diff
@@ -991,6 +991,7 @@ Architecture: linux-any
 Section: libs
 Depends: ${misc:Depends},
          ${shlibs:Depends},
+Build-Profiles: <pkg.ceph.jaeger>
```

Comment:
could you prefix the title of your commit message with the subcomponent your are changing ? see https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#3-describe-your-changes

Reply:
"debian,install-deps" would suffice.